main/Accordion/AccordionItem.tsx

```diff
@@ -40,37 +40,36 @@ type ToggleIndicatorType = React.ReactElement | undefined;
 interface TranslateYType {
   translateY: Animated.Value;
 }
+
 interface Props {
   testID: string;
```

Comment:
We need additional `style` for referencing itself :)

This is useful to add styles like `margin` explicitly.

Reply:
Oh. I missed it. my bad. :)

Add it [here](https://github.com/dooboolab/dooboo-ui/pull/69/commits/b03a7bbadba412cb3144a912c247072d2db00c8c).